ajax/libs/webcomponentsjs/package.json

```diff
@@ -10,18 +10,18 @@
     "shim"
   ],
   "license": "BSD-3-Clause",
-  "autoupdate": {
```

Comment:
1) I think you should not update to npm, cause of low number of releases on npm then on git repo
2) just add "bundles/*" to git source array and I think it will enough  for get bundles folder from git repo

Reply:
Git has more releases than NPM because the project was originally distributed through Bower, and moved to NPM later.

Since one year ago the official installation method is through NPM, and all releases since that time have been published to NPM. I think NPM is the correct way to update the project.

If you still think it is better to update through git I will change the PR as requested.

Reply:
Bump!

I will change the PR to use git updating if that's better. I just thought the docs indicated that NPM updating was preferred. Either one is fine with me.